lib/foods.js

```diff
@@ -9,18 +9,12 @@ $(document).ready( function() {
 function setFoodListener() {
   $('#food-form').on('submit', function(event) {
     event.preventDefault();
-    let name = $('[name=food-name]').val();
-    let calories = $('[name=calories]').val();
-
-    $.ajax({
-      method: 'POST',
-      url: api + '/api/v1/foods',
-      data: {food: {name: name, calories: calories}},
-      success: function(data) {
-        alert('Added food!')
-        foodRows(data);
-      }
-    })
+    // let name = $.trim($('[name=food-name]').val());
+    // let calories = $.trim($('[name=calories]').val());
+    let name = $('[name=food-name]');
+    let calories = $('[name=calorie-amount]');
```

Comment:
if we aren't using `this` or `event.target` to select these values, is there a reason not to just find them in our function calls below? do we need to pass them in?